frontend/src/components/pages/organization/OrgShop.tsx

```diff
@@ -1,32 +1,76 @@
 import { useQuery } from "@apollo/client";
-import { Box, Grid, Stack, Typography } from "@mui/material";
+import { Box, Grid, Stack, Typography, Button} from "@mui/material";
+import {KeyboardArrowLeft, KeyboardArrowRight} from '@mui/icons-material';
+import { useTheme } from '@mui/material/styles';
 
 import { AdminOrganizationFragment, AllShopOrdersDocument } from "@/generated/graphql";
-
 import { ShopSale } from "../orgs/ShopSale";
+import React,{ useState } from 'react';
 
 type Props = {
   organization: AdminOrganizationFragment;
 };
 export const OrgProducts: React.FC<Props> = ({ organization }) => {
-  const { data, error } = useQuery(AllShopOrdersDocument);
+
+  const limit = 5
```

Comment:
Remember that when this is merged it will be in production so remember to change this from 5 to something like 100 before merging

Reply:
Done!